main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/spf13/viper"
 )
 
-const atlantisVersion = "0.9.0-cp"
+const atlantisVersion = "0.9.1-cp"
```

Comment:
If we are going to use `-cp`, which I like, then I think we should leave the patch level the same as what we forked, and call this release `0.9.0-cp.1`

Reply:
updated